rct229/data_fns/table_G3_4_fns.py

```diff
@@ -0,0 +1,79 @@
+import rct229
+from rct229.data import data
+from rct229.data_fns.table_utils import find_osstd_table_entry
+from rct229.schema.config import ureg
+
+# This dictionary maps the opaque surface type to construction enumerations to
+# the corresponding construction values in ashrae_90_1_prm_2019.construction_properties.json
+OPAQUE_SURFACE_TYPE_TO_CONSTRUCTION_MAP = {
+    "ABOVE-GRADE WALL": "PRM Steel Framed Exterior Wall",
+}
+
+# This dictionary maps the surface conditioning category enumerations to
+# the corresponding building category values in ashrae_90_1_prm_2019.construction_properties.json
+SURFACE_CONDITIONING_CATEGORY_TO_BUILDING_CATEGORY_MAP = {
```

Comment:
from get_surface_conditioning_category

Reply:
So maybe change line 12 to "This dictionary maps surface conditioning categories as returned from `get_surface_conditioning_category_dict()` to"